examples/systemcalls/systemcalls.c

```diff
@@ -80,20 +97,43 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
         command[i] = va_arg(args, char *);
     }
     command[count] = NULL;
-    // this line is to avoid a compile warning before your implementation is complete
-    // and may be removed
-    command[count] = command[count];
+    openlog("writer app", LOG_PID, LOG_USER);
 
 
-/*
- * TODO
- *   Call execv, but first using https://stackoverflow.com/a/13784315/1446624 as a refernce,
- *   redirect standard out to a file specified by outputfile.
- *   The rest of the behaviour is same as do_exec()
- *
-*/
+    syslog(LOG_INFO, "File %s opened successfully !\n", outputfile);
```

Comment:
@ashwinhr186 think about how you could avoid duplicating logic between these two functions.